README.md

```diff
@@ -39,8 +39,11 @@ this:
   security or orchestrator agents. **Please do not manage your containers through
   this feature.** 
 * `compose-environment` (optional, JSON object) [Environment variables for docker-compose][compose-env].
+* `azure-environment` (optional, string) Azure environment. Supported values: "AzureCloud", "AzureChinaCloud". 
+  Default is "AzureCloud". [Azure Regions][azure-env]
 
 [compose-env]: https://docs.docker.com/compose/reference/envvars/
+[azure-env]: https://azure.microsoft.com/en-us/regions/
```

Comment:
linking to the regions page is not relevant in this context. It doesn't help you about what are the supported values, it does not give you strings like `AzureCloud` or `AzureChinaCloud`. So I say it is more confusing then helpful here.